src/POGOProtos/Map/Pokemon/WildPokemon.proto

```diff
@@ -10,5 +10,5 @@ message WildPokemon {
 	double longitude = 4;
 	string spawn_point_id = 5;
 	.POGOProtos.Data.PokemonData pokemon_data = 7;
-	int32 time_till_hidden_ms = 11;
+	int64 time_till_hidden_ms = 11;
```

Comment:
Someone brought up the theory, that it's just the time difference between their server starting time and now, so it probably didn't overflow before and was just a normal negative value, I'd say. And after that time in september it'll just overflow again and repeat the cycle :P